HW04/vitaly_kadyrko/spec/mentor_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative '../lib/homework'
+require_relative '../lib/mentor'
+require_relative '../lib/student'
+require_relative '../lib/notification'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'Test', surname: 'Student') }
+  let(:mentor) { described_class.new(name: 'Test', surname: 'Mentor') }
+  let(:homework) { mentor.add_homework(title: 'Test title', description: 'Test description', student: student) }
+
+  context 'when add_homework method used' do
+    it 'return Homework class object' do
```

Comment:
Use Present Simple everywhere

```suggestion
    it 'returns Homework class object' do
```